packages/fast-usdc/src/exos/settler.js

```diff
@@ -338,7 +338,13 @@ export const prepareSettler = (
          */
         forward(txHash, fullValue, EUD) {
           const { settlementAccount, intermediateRecipient } = this.state;
-          const dest = chainHub.makeChainAddress(EUD);
+          let dest;
+          try {
+            dest = chainHub.makeChainAddress(EUD);
+          } catch (e) {
+            log('⚠️ forward transfer failed!', e, txHash);
+            return statusManager.forwarded(txHash, false);
+          }
```

Comment:
👍

Reply:
The mutable `dest` is a smell.

Not a blocker for this testing PR, but I think we need a more general error handling solution.

- https://github.com/Agoric/agoric-sdk/issues/10968

Also not a blocker but this change should not be part of a `test` commit. It's a runtime change, specifically a `fix`.

Reply:
> previously observed the stuck-at-observed behavior when attempting to write a bootstrap test

Was that bootstrap test hurdle due to the problem here? I.e. could we port these multichain-testing tests to bootstrap style and they'll work?

We might want to do that, to help uncover where errors are not handled correctly (eg where the test can't be written, like you ran into)

Reply:
> Could we port these multichain-testing tests to bootstrap style and they'll work?

I'm not sure unless I try. Maybe I can follow up after this PR?

Reply:
If that's the case let's wait until https://github.com/Agoric/agoric-sdk/issues/10811